src/components/logo/Diary.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import AllProductsList from 'components/productsList/allProductsList/AllProductsList';
+import DiaryAddProduct from 'components/DiaryAddProductForm/DiaryAddProductForm';
+
```

Comment:
This file is a duplicate of the Diary component but looks like it is not as to data and should probably be deleted?